pynitrokey/cli/nk3/test.py

```diff
@@ -79,7 +79,11 @@ def log_devices() -> None:
     logger.info(f"Found {len(ctap_devices)} CTAPHID devices:")
     for device in ctap_devices:
         descriptor = device.descriptor
-        logger.info(f"- {descriptor.path} ({descriptor.vid:x}:{descriptor.pid:x})")
+        if isinstance(descriptor.path, bytes):
+            path = descriptor.path.decode("cp1252")
```

Comment:
`cp1252` is a quite bold assumption. I doubt it will work outside Germany. Do we really need that?
Please extract treating coding to a helper method for a centralized handling.

Reply:
The string is returned by [`SetupDiGetDeviceInterfaceDetailA`](https://docs.microsoft.com/en-us/windows/win32/api/setupapi/nf-setupapi-setupdigetdeviceinterfacedetaila).  According to [this page](https://docs.microsoft.com/en-us/windows/win32/learnwin32/working-with-strings), the `A` suffix indicates “an ANSI string“.  [Apparently](https://stackoverflow.com/questions/701882/what-is-ansi-format), `cp1252` is the most common meaning of this term.  Is there a better way to solve this without patching fido2 to use the `W` variant instead?

Reply:
Comment below some said its `windows-1254` for him. Another example is Polish edition, which has most probably `cp1250` as the default code page. It might work anyway, since there should not be any special characters, but I do not know that. What are other options? Can we treat it as raw ASCII and bail otherwise?

Reply:
As discussed directly, we decided to use ISO 8859-1 which is the superset of all possible code pages and ignore decoding errors as the string is expected to be ASCII and only used for display and device selection.